lib/cartopy/mpl/geoaxes.py

```diff
@@ -784,6 +796,187 @@ def stock_img(self, name='ne_shaded'):
         else:
             raise ValueError('Unknown stock image %r.' % name)
 
+    def background_img(self, name='ne_shaded', resln='low', extent=None,
+                       cache=False):
+        """
+        Adds from a selection of prepared images to the map where the
+        available files are held in the CARTOPY_USER_BACKGROUNDS environment
+        variable.
+
+        That directory is checked with read_user_backgound_images and needs to
+        contain a JSON file which defines for the image metadata.
+
+        Kwargs:
+
+            * name - the name of the image to read according to the contents of
+                     the JSON file. A typical file might have, for instance:
+                     'ne_shaded' : Natrual Earth Shaded Relief
+                     'ne_grey' : Natural Earth Grey Earth
+
+            * resln - the resolution of the image to read, according to the
+                      contents of the JSON file. A typical file might have
+                      the following for each name of the image:
+                      'low', 'med', 'high', 'vhigh'.
```

Comment:
'full' is possibly better than 'vhigh' because it corresponds with GSHHS (though that's not raster).